builtins.go

```diff
@@ -373,6 +374,33 @@ func builtinType(e *evaluator, xp potentialValue) (value, error) {
 	return makeValueString(x.typename()), nil
 }
 
+// TODO(sbarzowski) not entierly clear what this is for
+const codepointMax = 0x110000
+
+func builtinChar(e *evaluator, xp potentialValue) (value, error) {
+	x, err := e.evaluateNumber(xp)
```

Comment:
Unicode allots 21bits for codepoints https://en.wikipedia.org/wiki/Code_point

Reply:
Proper comment added.